rosdep/python.yaml

```
@@ -3018,6 +3018,14 @@ python-pathlib:
   debian: [python-pathlib]
   gentoo: [dev-python/pathlib]
   ubuntu: [python-pathlib]
+python-pathlib2:
+  arch: [python2-pathlib2]
+  debian: [python-pathlib2]
+  fedora: [python-pathlib2]
+  gentoo: [dev-python/pathlib2]
+  ubuntu: 
+    '*': [python-pathlib2]
+    xenial: null
```

Fix lexical sorting of Ubuntu definitions

```suggestion
  ubuntu: 
    xenial: null
    '*': [python-pathlib2]
```

---

I'm a bit confused because the current order (first '*', then xenial) is already being used in other places https://github.com/ros/rosdistro/blob/074cafd4261f88aaab8b11c4592b2b017732e5c3/rosdep/python.yaml#L216-L217.

---

I don't think this is the issue.  I think the problem is trailing whitespace after `ubuntu:`.  I'll push a fix.